tools/pushDockerImageToECR.sh

```diff
@@ -51,9 +51,15 @@ function prepare_artifacts() {
 EOF
 
 	echo "==> Downloading artifact for tag '$TAG' from releases"
-	local artifact_name="exasol-glue-connector-$TAG-assembly.jar"
-	curl -JLO "https://github.com/exasol/glue-connector/releases/download/$TAG/$artifact_name"
-	mv "$artifact_name" artifacts/
+	local artifact_jar="exasol-glue-connector-$TAG-assembly.jar"
+	local artifact_sha="exasol-glue-connector-$TAG-assembly.jar.sha256"
+	curl -JLO "https://github.com/exasol/glue-connector/releases/download/$TAG/$artifact_jar"
+	curl -JLO "https://github.com/exasol/glue-connector/releases/download/$TAG/$artifact_sha"
+	mkdir -p target/
```

Comment:
It would be more robust to use [`mktemp`](https://www.gnu.org/software/autogen/mktemp.html) in situations like this.